robusta_krr/formatters/table.py

```diff
@@ -20,12 +21,40 @@ def _format(value: RecommendationValue) -> str:
         return resource_units.format(value)
 
 
+def __calc_diff(allocated, recommended, selector, multiplier=1) -> str:
+    if recommended is None or isinstance(recommended, str) or selector != "requests":
```

Comment:
you only want to calculate the difference for requests and not limits?

Reply:
Yes, [here](https://docs.google.com/spreadsheets/d/1NUhBiKzsPioI8xyAOzDej3C7bomjPQnVJTb0izCuJcQ/edit#gid=1035852509) is the link for the spec

Reply:
cool, just wanted to make sure, was confused that i didnt see it for limits